cmd/autotestgridgenerator/main.go

```diff
@@ -98,9 +99,26 @@ func main() {
 		logrus.Info("no new commits, exiting ...")
 		os.Exit(0)
 	}
+	newRepo := githubRepo
+	gc.SetMax404Retries(0)
+	if _, err := gc.GetRepo(o.githubLogin, githubRepo); err != nil {
+		// Somehow github.IsNotFound doesn't recognize this?
+		if !strings.Contains(err.Error(), "status code 404") {
+			logrus.WithError(err).Fatalf("unexpected error when getting repo %s/%s", o.GitName, githubRepo)
+		}
+		logrus.Info("Creating fork")
```

Comment:
nit: logging *why* (it does not exist) and logging more (creating fork of x/y) coyuld be helpful